backend/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.2'
+
+services:
+  versewise-cms-backend:
+    build:
+      context: .
+      network: host
+    ports:
+      - "5000:5000"
```

Comment:
Better to read the host port from .env file "Host_Port:Container_Port", it's okay to  leave container port as it's.
- [ ] use .env for host port